tests/integration-tests/configs/schedulers.yaml

```diff
@@ -136,6 +136,12 @@ test-suites:
           instances: ["c5.xlarge"]
           oss: ["alinux2"]
           schedulers: ["slurm"]
+    test_slurm.py::test_slurm_ticket_17399:
```

Comment:
Since you changed the name of the test you should also change the name of the subfolder with the config file, otherwise the test will fail